### PR TITLE
[DOCFIX] Update Reference doc by running alluxio docGen

### PR DESCRIPTION
**Pull request by LuQQiu**
Update the metrics and configuration in reference docs

### CODE REVIEW

File: docs/_data/table/en/client-metrics.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
-Client.BlockReadDataChunk:
+Client.BlockReadDataChunkDirect:
   'The timer statistics of reading block data in chunks from Alluxio workers. This metrics will only be recorded when alluxio.user.block.read.metrics.enabled is set to true'
-Client.BlockReadDataFromChunk:
+Client.BlockReadDataChunkRemote:
```
Comment (apc999) [inline docs/_data/table/en/client-metrics.yml:3]: @LuQQiu  I think this metrics `Client.BlockReadDataChunkRemote` is only introduced in https://github.com/Alluxio/alluxio/pull/13438/, which has not been merged , right?

**Review by apc999 — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

File: docs/_data/table/en/client-metrics.yml [hunk 2]
```diff
@@ -64,6 +64,8 @@ Client.CachePutStoreDeleteErrors:
   'Number of failures when putting cached data in the client cache due to failed deletes in page store.'
 Client.CachePutStoreWriteErrors:
   'Number of failures when putting cached data in the client cache due to failed writes to page store.'
+Client.CachePutStoreWriteNoSpaceErrors:
+  'Number of failures when putting cached data in the client cache but getting disk is full while cache capacity is not achieved. This can happen if the storage overhead ratio to write data is underestimated.'
```
Comment (Limuzhou) [inline docs/_data/table/en/client-metrics.yml:68]: Is "disk is full" an error message?

**Review by Limuzhou — COMMENTED** [review]
Minor confusion

**Pull request merged**